rust/cymbal/src/main.rs

```diff
@@ -51,12 +49,12 @@ fn start_health_liveness_server(config: &Config, context: Arc<AppContext>) -> Jo
 }
 
 #[tokio::main]
-async fn main() -> Result<(), Error> {
+async fn main() {
     setup_tracing();
     info!("Starting up...");
 
-    let config = Config::init_from_env()?;
-    let context = Arc::new(AppContext::new(&config).await?);
+    let config = Config::init_from_env().unwrap();
+    let context = Arc::new(AppContext::new(&config).await.unwrap());
```

Comment:
I'm guessing unwraps are preferred here because it is basically impossible?

Reply:
Basically, I'd rather just fail to start up than let callers try and handle this error (it just gets returner by main anyway, which then panics).